framework/wazuh/ossec_socket.py

```diff
@@ -23,30 +24,30 @@ def _connect(self):
             self.s = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
             self.s.connect(self.path)
         except Exception as e:
-            raise WazuhException(1013, str(e))
+            raise WazuhInternalError(1013, str(e))
```

Comment:
We should use explicit kwargs where available, in this case I think you need to use the `extra_message`, right?

Reply:
Fixed